css/legend_info.css

```diff
@@ -27,5 +27,22 @@
     background-color: #4e4e4e;
     box-shadow: 0 0 15px rgba(0,0,0,0.2);
     border-radius: 10px;
-    
+
+}
+
+.leaflet-control-layers-expanded {
```

Comment:
Hy @lynnnakiganda , can you remove these css changes, please. 
I do not see them being requested for in the change sheet and I personally prefer the sidepanel with the shadows.